package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "airbnb-prop-types": "^2.10.0",
     "brcast": "^2.0.2",
-    "deepmerge": "^1.5.2",
```

Comment:
Merely being outdated is not inherently bad; in this case, why not v4?

Reviewing the [changelog](https://github.com/TehShrike/deepmerge/blob/master/changelog.md) suggests that the `clone` option now defaults to `true`, which might affect [our usage](https://github.com/airbnb/react-with-direction/blob/0b6625cb9979514bb0042ffc6487bc5203e69054/src/withDirection.jsx#L71), but otherwise it looks safe.